service/src/main/java/org/databiosphere/workspacedataservice/config/DataImportProperties.java

```diff
@@ -22,6 +22,7 @@ public class DataImportProperties {
   private RecordSinkMode batchWriteRecordSink;
   private String rawlsBucketName;
   private boolean succeedOnCompletion;
+  private boolean permissionSync;
```

Comment:
Initializing to false would allow omitting this from the data plane profile.

```suggestion
  private boolean permissionSync = false;
```

Reply:
ha thanks that's great, changed